src/sql-editor/standardSql/standardSuggestionsRegistry.ts

```diff
@@ -48,6 +54,39 @@ export const initStandardSuggestions =
             },
           ]),
       },
+      {
+        id: SuggestionKind.SelectMacro,
+        name: SuggestionKind.SelectMacro,
+        suggestions: (_, m) =>
+          Promise.resolve([
+            ...macros
+              .list()
+              .filter((m) => m.type === MacroType.Value)
```

Comment:
Let's add a column macro suggestion here. Will mostly be supported looking at i.e. athena or redshift data source. For BQ we gonna return an error about this macro not being supported.

```suggestion
              .filter((m) => m.type === MacroType.Value || m.type === MacroType.Column)
```